packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -442,7 +445,11 @@ abstract class ResidentRunner {
   String get projectRootPath => _projectRootPath;
   String _mainPath;
   String get mainPath => _mainPath;
-  String getReloadPath({bool fullRestart}) => mainPath + (fullRestart ? '' : '.incremental') + '.dill';
+
+  String getReloadPath({bool fullRestart, @required bool trackWidgetCreation}) {
```

Comment:
I don't quite understand why this is used the way it is. mainPath is typically the path to a dart file not really a dill file but somehow this works. not sure if this can be simplified or cleaned up somehow.

Reply:
From what I remember, this is file name that is given to the vm on the device for reload.